support/modules/rootfs-image

```diff
@@ -69,21 +85,21 @@ EOF
 
     ArtifactVerifyReboot)
         ensure_correct_root_mounted
-        if test "$(${PRINTENV} upgrade_available)" != "upgrade_available=1"; then
+        if test "$upgrade_available" != 1; then
```

Comment:
Nit, if right side is not quoted, then either should the left side have to be?

Reply:
A variable should always be quoted, because it could have unexpected content, unintentionally. The right side could also be quoted, of course, but it's a constant, so there is no need. No strong preference on whether it should be, I just thought it looked nicer.

Reply:
I'm aware, I just struck me that we're essentially doing string comparison with one string (I know sh don't care what's there)